src/components/cards/VariantCard.tsx

```diff
@@ -184,26 +184,40 @@ const getDiseaseListForFeatures = (features: VariantData[]) => {
   return diseaseSet;
 };
 
-const getFilters = (data: VariantData[]) => {
-  const diseases = getDiseaseListForFeatures(data);
-
-  return Array.from(diseases).map((diseaseName) => {
-    return {
-      name: diseaseName as string,
-      type: { name: "diseases", text: "Disease" },
-      options: { labels: [diseaseName as string], colors: ["#A31D5F"] },
-    };
-  });
-};
-
 const VariantCard: FunctionComponent<{ data: VariationData }> = ({ data }) => {
   const id = v1();
```

Comment:
Same thing here, I saw where it's used lower in the code, but I'd try to make it work with something unique from the VariationData.
If that can't be done, use a ref:
```
const idRef = useRef(v1());
// then lower in the code
`[data-uuid='${idRef.current}-manager']`
```
because otherwise the value of `id` will change at each render